packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -275,11 +269,10 @@ const ProgramCard = ({
       </IconWrapper>
       <DurationWrapper dir={dir} dateTime={duration}>
         <VisuallyHiddenText>
-          {` ${durationLabel} ${formatDuration({
-            duration,
-            format: getDurationFormat(duration, ','),
-            locale,
-          })} `}
+          {`${detokenise(
+            ` ${durationLabel} %duration%`,
```

Comment:
Could you pass in the duration label so it is the whole string, including the token? Different languages will have the token in different places :)

Reply:
Yes, sorry, this was in place just so that duration would work with what we have in Simorgh now (since we don't have proper translations with %duration% yet). But instead we can change this line + make this pr a major change + only pull it into Simorgh when translations are ready :)

Reply:
Thanks! I think what we might have to do is update the translations in Simorgh to include tokens based on what we already have, and then update them with the token in the correct place once we get the translations back.